v_0_1_0/master_template.go

```diff
@@ -1545,7 +1549,7 @@ write_files:
       for manifest in $MANIFESTS
       do
           while
-              /usr/bin/docker run --net=host --rm -v /srv:/srv $KUBECTL apply -f /srv/$manifest
+              /usr/bin/docker run -e KUBECONFIG=${KUBECONFIG} --net=host --rm -v /srv:/srv /etc/kubernetes:/etc/kubernetes $KUBECTL apply -f /srv/$manifest
```

Comment:
you missed `-v` before the `/etc/kubernetes` volume on each line

Reply:
done